App.tsx

```diff
@@ -14,9 +15,11 @@ export default function App() {
     return null;
   }
   return (
-    <SafeAreaProvider>
-      <Navigation colorScheme={colorScheme} />
-      <StatusBar />
-    </SafeAreaProvider>
+    <PaperProvider>
```

Comment:
I made these changes in a separate PR that is merged, can you pull from master and resolve any conflicts? #17